examples/gno.land/p/demo/microblog/microblog.gno

```diff
@@ -0,0 +1,174 @@
+package microblog
+
+import (
+	"errors"
+	"std"
+	"strings"
+	"time"
+
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/ufmt"
+)
+
+type Microblog struct {
+	Title  string
+	Prefix string   // i.e. r/gnoland/blog:
+	Pages  avl.Tree // author (string) -> Page
+}
+
+func NewMicroblog(title string, prefix string) (m *Microblog) {
+	return &Microblog{
+		Title:  title,
+		Prefix: prefix,
+		Pages:  avl.Tree{},
+	}
+}
+
+func (m *Microblog) GetPages() []*Page {
+	var (
+		pages = make([]*Page, m.Pages.Size())
+		index = 0
+	)
+
+	m.Pages.Iterate("", "", func(page *avl.Node) bool {
+		pages[index] = page.Value().(*Page)
+		index++
+		return false
+	})
+
+	// TODO: sort by last updated
```

Comment:
Leftover TODO?

Reply:
Good catch! I implemented the `sort.Sort` to make this happen.